octodns_ultra/__init__.py

```diff
@@ -350,6 +350,21 @@ def populate(self, zone, target=False, lenient=False):
         )
         return exists
 
+    def _force_root_ns_update(self, changes):
+        '''
+        Changes any 'Create' changetype for a root NS record to an 'Update'
+        changetype. Used on new zone creation, since Ultra will automatically create root NS records.
+        This means our desired NS records must be applied as an Update, rather than a Create.
+        '''
+        for change in changes:
+            if (
+                change.record.name == ""
+                and isinstance(change.record, NsRecord)
```

Comment:
most places look at the `_type` field rather than class, mostly recommend that for consistency.

```suggestion
                and change.record._type == 'NS'
```